Runs: Rewrite Runs section (#2011)
wandb/docs: models/runs.mdx

Reviewer: Now reviewed up to here.

A *run* is a single unit of computation logged by W&B. You can think of a W&B Run as an atomic element of your whole project. In other words, each run is a record of a specific computation, such as training a model and logging the results, hyperparameter sweeps, and so forth.

> Common reasons for initiating a run include, but are not limited to: 

Reviewer: ```suggestion
Some common reasons for initiating a run include: 
```
But this seems like kind of a weird way to talk about this.

A *run* is a single unit of computation logged by W&B. You can think of a W&B Run as an atomic element of your whole project. In other words, each run is a record of a specific computation, such as training a model and logging the results, hyperparameter sweeps, and so forth.

> Some common reasons for initiating a run include: 

Reviewer: I keep feeling like this is a strange phrasing. Maybe "Start a run to: - Train a model...."
